File: homework/hw3/Hw3_thing.java


public class Hw3_thing {
    public int a;

    public static void main(String[] argv) {
        Hw3_thing b = new Hw3_thing();
        Hw3_thing c;

        b.a = 5;
        c = b;
        b.a = 4;

        System.out.println("" + b.a);
        System.out.println("" + c.a);
    }
}

File: homework/hw3/hw3_thing.c
#include <printf.h>

typedef struct {
    int a;
} Thing;

int main(int argc, char *argv[]) {
         Thing b = {0};
         Thing c;

         b.a = 5;
         c = b;
         b.a = 4;

         printf("%d\n", b.a);
         printf("%d\n", c.a);

    return 0;
}
